onboarding.md

```diff
@@ -31,293 +30,27 @@ That is it! Go to your onboarding issue and start checking things off!
-
-
-Regards
-```
-
-
-
-# [Email Template] Niteo Email Account
-
-```
-Hey!
-
-Here's your Niteo email. You'll be using it for communication with us and for subscribing to websites.
-
-w: https://apps.rackspace.com/
-u: [email]
-p: [pass]
-
-Please change your password immediately after you login. You can do that in the top right Settings menu.
-
-Configure the email on your desktop app, find the instructions here: https://emailhelp.rackspace.com
-
-The next thing that you need to do is set up S/MIME email encryption. Generate a Comodo Certificate at https://www.comodo.com/home/email-security/free-email-certificate.php. You’ll need to Google on how to set encryption on your desktop app.
-
-This email is signed with my cert, you should be able to reply with an encrypted message, signed with your cert so that I can then reply back with an encrypted message as well.
-
-Once you have encryption ready, I’m sending over more information.
-
-
-Regards
-```
-
-
-# [Email Template] OpenVPN Access
-
-```
-Hey!
-
-Now that encryption is configured, you need to configure OpenVPN so you can connect to our internal services.
-
-Configuration files are attached, normally you just double-click them and follow instructions.
-
-When you manage to connect to our VPN you will then be able to access our internal services, such as http://docs.niteoweb.com/.
-
-
-Regards
-```
-
-# Email Signatures
-
-A few examples to base your signature off of. Choose your own title, it can be whatever you wish.
-
-```
-Richard Hendricks
-
-CEO
----------------
-Niteo.co
-
-E: rh@niteo.co
-W: www.niteo.co
-```
-
-```
-Erlich Bachmann
-
-Backend Specialist
-------------------
-Niteo.co
-
-E: eb@niteo.co
-W: www.niteo.co
-```
-```
-Monica Hall
-
-Senior (con)Sultan
-------------------
-Niteo.co
-
-E: mh@niteo.co
-W: www.niteo.co
-```
-
-# Offboarding
-
-```
-* [ ] Access to team 1Password.
-* [ ] Heroku access.
-* [ ] GitHub repositories access.
-* [ ] PyPICloud access.
-* [ ] Donations spreadsheet.
-* [ ] Slack access.
-* [ ] Email account.
-* [ ] Intra account.
-* [ ] VPN access.
-* [ ] Zoom.us account.
-* [ ] Sentry access.
-* [ ] EBN account.
-* [ ] DMON account.
-```
+* Add the new team member to the [@niteoweb/trialists] group so they get access to [operations](https://github.com/niteoweb/operations) and [handbook](https://github.com/niteoweb/handbook) repositories.
+* Create an [Trialist Onboarding Issue] and assign it to the new team member.
+* Send the welcome email, using the [Welcome  email template].
+* Create a `@niteo.co` email address and send credentials using the [Niteo account email template].
+* Send the OpenVPN details using the [OpenVPN access email template].
+
+Further useful template links:
+* [Intellectual Property Rights Disclaimer]
+* [Full-time Onboarding Issue]
+* [Offboarding Issue].
+
+<!-- References --->
+
+[@niteoweb/trialists]: https://github.com/orgs/niteoweb/teams/trialists
+[Welcome email template]: templates/onboarding-emails.md#welcome
```

Comment:
Can we make this links `mailto:` links that would automatically open up my email client and populate the subject and body fields? That would be just pure magic :)

Reply:
Like this... https://github.com/niteoweb/handbook/blob/03afe878a09d1921fce45a98ae917b56b60e20fc/onboarding.md#peopleops-checklist